style.css

```diff
@@ -471,3 +479,396 @@ h2 {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headi {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 72px;
+    font-weight: 400;
+    line-height: 82px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-left: 235px;
+    width: 566px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: -60px;
+    display: flex;
+    flex-direction: row;
+    margin-left: 153px;
+  }
+
+  .line {
+    width: 794px;
+    height: 0;
+    margin-top: 150px;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 185px;
+    margin-top: 70px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(./Images/Img-Placeholder4.png);
+    background-repeat: no-repeat;
+    height: 448px;
+    width: 684px;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    justify-items: center;
+    align-items: center;
+  }
+
+  .art-container {
+    height: 435px;
+    width: 369px;
+    margin-left: 142px;
+    background-image: url("./Images/Img-Placeholder10.png");
+  }
+
+  .art-container:hover {
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+
+  .art-container1 {
+    height: 435px;
+    width: 369px;
+    margin-left: 10px;
+    background-image: url("./Images/Img-Placeholder11.png");
+  }
+
+  .art-container2 {
+    height: 435px;
+    width: 369px;
+    margin-right: 143px;
+    background-image: url("./Images/Mask-Group.png");
+  }
+
+  .v2 {
+    padding: 12px;
+  }
+
+  .button {
+    margin-top: 49px;
+  }
+
+  .about {
+    width: 100%;
+    margin-top: 330px;
+    margin-left: 0;
+  }
+
+  .about-me-footer {
+    height: 582px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .image-bg {
+    display: flex;
+    flex-direction: row;
+    margin-right: 14%;
+    background-image: url("./Images/illustration-about\ me@3x.png");
+    background-size: 310px 268px;
+    background-position: 10% 95%;
+    transform: rotate(360deg);
+  }
```

Comment:
- [ ] kindly, make sure the background of the about section match the figma design. The background image seem have a gap with the right side of the screen. 

**Your project design**
![image](https://user-images.githubusercontent.com/74411135/236283092-6b80bc00-db10-4668-8159-b60d52ee75a7.png)

**The figma design**
![image](https://user-images.githubusercontent.com/74411135/236283224-1c3799d4-18c5-4937-889f-8262895d74ff.png)

That will make your project look way better as the Figma design is done by a professional designer 😮 !